### PR TITLE
Updated Utils class $ started TC145

### PR DESCRIPTION
**Pull request by asag0**

### CODE REVIEW

File: src/utils/MainUtils.java [hunk 1]
```diff
@@ -3,4 +3,14 @@
 public class MainUtils {
 
 
+    public void coolMethod(){
```
Comment (mcorasag) [inline src/utils/MainUtils.java:6]: Cool methods, but they should be static

**Comment by asag0** [comment]
You're right, good catch. Made them static. GLM

**Review by mcorasag — COMMENTED** [review]
Great methods, Looks good!

**Review by mcorasag — APPROVED** [review]
Looking good!

**Pull request merged**